apps/app/repository.tf

```diff
@@ -13,25 +13,27 @@
 #  limitations under the License.
 
 resource "github_repository" "app" {
+  count       = local.create_repo ? 1 : 0
   name        = var.app_id
   description = var.app_config.description
   visibility  = var.app_config.managed_repo.private ? "private" : "public"
 
   template {
-    owner      = "UCLH-Foundry"
-    repository = var.app_config.managed_repo.template
+    owner      = split("/", var.app_config.managed_repo.template)[0]
+    repository = split("/", var.app_config.managed_repo.template)[1]
   }
 }
 
 resource "github_team" "owners" {
+  count       = local.create_repo ? 1 : 0
```

Comment:
Related to another comment - we could potentially abstract all the resources just for creating a repo into a sub-module and have a single flag, but again not something we need to do now